packages/cf-builder-pagination/src/PaginationBuilder.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Pagination, PaginationItem } from 'cf-component-pagination';
-import Icon from 'cf-component-icon';
+import Icon from '../../cf-component-icon/src';
```

Comment:
Hm, yeah, this is interesting. Is the only reason we need this to make tests pass? Can't we reference a specific version of `cf-component-icon` in `cf-builder-pagination`?